keel-sql/src/test/kotlin/com/netflix/spinnaker/keel/tx/DeliveryConfigTransactionTests.kt

```diff
@@ -49,6 +50,7 @@ import strikt.assertions.isFalse
     "spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver"
   ]
 )
+@DisableSpringScheduling
 internal class DeliveryConfigTransactionTests : JUnit5Minutests {
```

Comment:
Scheduling was enabled here and I was seeing a bunch of noise from calls to mocked repositories